lib/fluent/plugin/output.rb

```diff
@@ -185,13 +185,26 @@ def initialize
         @simple_chunking = nil
         @chunk_keys = @chunk_key_time = @chunk_key_tag = nil
         @flush_mode = nil
+        @timekey_zone = nil
       end
 
       def acts_as_secondary(primary)
         @as_secondary = true
         @primary_instance = primary
+
+        @chunk_keys = @primary_instance.chunk_keys if @primary_instance.chunk_keys
```

Comment:
`@chunk_keys` will not be initialized when `@primary_instance.chunk_keys` is not specified. It will be warned.